### DISCUSSION TITLE
CP Listing: show relationships and format columns

### DISCUSSION THREAD

**Discussion by jorisvan-ons** [statamic-rad-pack/runway#230] [Help] [answered]
Hi, 

What are all the config options for showing related collections/models and how do I specify how a certain value should be formatted (for example transform a complete datetime value into a date) on the CP listing?

Maybe the documentation can be extended a little bit to accomodate this?

**Comment by duncanmcclean** [comment, marked as answer]
Hey!

I've just left [a comment](https://github.com/duncanmcclean/runway/discussions/227#discussioncomment-5181529) on your other discussion where you've asked a similar-ish question.

I'll see what I can do in terms of improving the documentation around this for others. 

> for example transform a complete datetime value into a date

In this particular case, you shouldn't need to format this yourself. Can you try adding the `time_enabled: false` config setting to your Date field and see if that helps?

Maybe also try ensuring the field is cast to `date`, rather than `datetime` in your Eloquent model.
